backend/src/src-licenseinfo/pom.xml

```diff
@@ -59,6 +71,18 @@
             <artifactId>saxon-dom</artifactId>
             <version>8.7</version>
         </dependency>
+        <dependency>
+            <groupId>org.dom4j</groupId>
+            <artifactId>dom4j</artifactId>
```

Comment:
dom4j has a custom license and one should look into its impliations: https://github.com/dom4j/dom4j/blob/master/LICENSE (this is some variant based on BSD4)

Reply:
There is not jet a CQ for version 2.1.1, but version 2.1.0 seems to be approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15713